aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/params.h

```diff
@@ -274,6 +280,18 @@ typedef void (*pytorch_q8gemm_ukernel_function)(
     size_t c_stride,
     const union pytorch_qnnp_conv_quantization_params* quantization_params);
 
+typedef void (*pytorch_q8gemm_dq_ukernel_function)(
```

Comment:
Can you add a comment here briefly describing what this kernel does

Reply:
Thanks, done.